apps/ui/src/components/Modal/DelegationConfig.vue

```diff
@@ -14,6 +14,7 @@ const DEFAULT_FORM_STATE = {
 
 const props = defineProps<{
   open: boolean;
+  availableDelegationTypes: DelegationType[];
```

Comment:
As we are reverting I think we can revert all related changes (remove availableDelegationTypes here and in parent component, `EDITOR_DELEGATION_TYPES` as well).